packages/expo-camera/src/WebCameraUtils.ts

```diff
@@ -377,7 +377,7 @@ export function setVideoSource(
   video: HTMLVideoElement,
   stream: MediaStream | MediaSource | Blob | null
 ): void {
-  const createObjectURL = window.URL.createObjectURL ?? window.webkitURL.createObjectURL;
+  const createObjectURL = window?.URL.createObjectURL ?? window.webkitURL.createObjectURL;
```

Comment:
```suggestion
  const createObjectURL = window?.URL.createObjectURL ?? window?.webkitURL.createObjectURL;
```
If `window` is optional then it should rather be that way 🤔

Reply:
I will revert as this was not the correct fix, it was caused by something else that has since been updated